avr/bootloaders/HoodLoader2/Board/Board.h

```diff
@@ -57,6 +57,7 @@ extern "C" {
 			#define ARDUINO_MICRO_PID   		0x0037 // Bootloader, not program!
 			#define ARDUINO_DUE_PID             0x003D
 			
+			#ifndef USB_DESCRIPTOR_STRING // If the Makefile did not supply a product string...
```

Comment:
What are you using this for? If no "normal" VID/PID is recognized it will use "Lufa" as string. I think there is an else path missing.